go/transactions/README.md

```diff
@@ -102,24 +102,24 @@ Each object in the query-signatures array represents a generalized query and inc
  * op: The operation type (e.g., "insert", "update", "delete").
  * affected_table: The table affected by the query.
  * updated_columns: (Only for update operations) An array of column names that are updated by the query.
- * predicates: An array of conditions (also known as predicates) used in the query’s WHERE clause. Each predicate abstracts the condition to focus on the pattern rather than specific values. Not all predicates are included in the query signature; only those that could be used by the planner to select if the transaction is a single shard or a distributed transaction.
+ * Predicates: An array of conditions (also known as Predicates) used in the query’s WHERE clause. Each predicate abstracts the condition to focus on the pattern rather than specific values. Not all Predicates are included in the query signature; only those that could be used by the planner to select if the transaction is a single shard or a distributed transaction.
```

Comment:
```suggestion
 * predicates: An array of conditions (also known as predicates) used in the query’s WHERE clause. Each predicate abstracts the condition to focus on the pattern rather than specific values. Not all predicates are included in the query signature; only those that could be used by the planner to select if the transaction is a single shard or a distributed transaction.
```